yamllint/rules/quoted.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018 ClearScore
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Use this rule to forbid any values that are not quoted. You can also enforce
+the type of the quote used - single or double.
+
+.. rubric:: Examples
+
+#. With ``quoted: {quote-type: any}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    foo: "bar"
+    bar: 'foo'
+
+   the following code snippet would **FAIL**:
+   ::
+
+    foo: bar
+    bar: 123
```

Comment:
Adding `123` in the *failure* example is a bit confusing... I'd rather have documentation as clear as possible, for instance:

**PASS**:
```yaml
foo: "bar"
bar: 'foo'
number: 123
boolean: true
```

**FAIL**:
```yaml
foo: bar
```